examples/hack/healthcheck.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"log"
+	"net/http"
+	"os"
+)
+
+func main() {
+	if len(os.Args) != 2 {
+		log.Fatalf("usage: %s <url>", os.Args[0])
+	}
+	url := os.Args[1]
+
+	r, err := http.Get(url)
```

Comment:
The default `http.Client` does not have a timeout. I'm not sure if there would be cases this could hang indefinitely, would be safer to construct a client with a default timeout of 5s.

Reply:
Makes sense - the docker compose.yaml also provides a timeout, so I assume docker would handle killing the healthcheck process - but not much harm having it in two places.

Reply:
Will set the timeout to 10s to match the compose file.

Reply:
update - just made a configurable timeout